packages/puppeteer-core/src/bidi/Browser.ts

```diff
@@ -323,7 +323,12 @@ export class BidiBrowser extends Browser {
     return this.#browserTarget;
   }
 
-  override disconnect(): void {
-    this;
+  override async disconnect(): Promise<void> {
+    try {
+      await this.#connection.send('session.end', {});
+      this.#connection.dispose();
+    } catch (e) {
+      console.log(e);
```

Comment:
I assume we don't want to show error here so we should use `debugError` to print only when explicitly enabled.

Reply:
done